src/components/Header.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FaGrin } from 'react-icons/fa';
 
 function Header() {
   return (
     <nav>
       <h1>Bookstore CMS</h1>
       <div className="link">
-        <li><Link to="/" style={{ textDecoration: 'none' }}>Books</Link></li>
-        <li><Link to="/Categories" style={{ textDecoration: 'none' }}>Categories</Link></li>
+        <li><Link to="/" style={{ textDecoration: 'none' }}>BOOKS</Link></li>
+        <li><Link to="/Categories" style={{ textDecoration: 'none' }}>CATEGORIES</Link></li>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement correct colors and use color effects(`Pressed` and `Enebled`) like below:

![image](https://user-images.githubusercontent.com/63915024/208978051-090cb6bd-0b8e-4380-a448-48f6bc1d1b8d.png)